### PR TITLE
tidb: add explain test

### PR DESCRIPTION
**Pull request by purelind**
* add explain test to VerfiyCi
releated to https://github.com/pingcap/tidb/pull/31853

### CODE REVIEW

**Comment by purelind** [comment]
replay test ci:
1. https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/62496/pipeline
2. https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/63810/pipeline/

File: jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy [hunk 1]
```diff
@@ -240,10 +240,10 @@ try {
             }
         }
 
-        if (notRun == 0){
-            println "the ${ghprbActualCommit} has been tested"
-            throw new RuntimeException("hasBeenTested")
-        }
+        // if (notRun == 0){
```
Comment (dianqihanwangzi) [inline jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy:243]: why comment out this code?

Comment (purelind) [reply jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy:243]: debug only...

**Review by dianqihanwangzi — APPROVED** [review]
lgtm

**Review by tangenta — APPROVED** [review]

**Pull request merged**